scantext/forms.py

```diff
@@ -0,0 +1,42 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# http://nexb.com and https://github.com/nexB/scancode.io
+# The ScanCode.io software is licensed under the Apache License version 2.0.
+# Data generated with ScanCode.io is provided as-is without warranties.
+# ScanCode is a trademark of nexB Inc.
+#
+# You may not use this software except in compliance with the License.
+# You may obtain a copy of the License at: http://apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+#
+# Data Generated with ScanCode.io is provided on an "AS IS" BASIS, WITHOUT WARRANTIES
+# OR CONDITIONS OF ANY KIND, either express or implied. No content created from
+# ScanCode.io should be considered or used as legal advice. Consult an Attorney
+# for any legal advice.
+#
+# ScanCode.io is a free software code scanning tool from nexB Inc. and others.
+# Visit https://github.com/nexB/scancode.io for support and download.
+
+from django import forms
+
+
+class LicenseForm(forms.Form):
+    input_text = forms.CharField(
```

Comment:
We may want to disable the default `strip` on this field because we are loosing some of the initial format when displaying the text in the results view.
See the first line for example:
![Screenshot 2022-06-21 at 11 07 25](https://user-images.githubusercontent.com/489057/174737653-d4671e44-b0bb-43a3-b5ee-ded34d9179af.jpg)

What's you take on this?

Reply:
We can set it to false in the form but the get_license does strip the matched_text, which means the card which shows detected text does look the same way above